charts/opentelemetry-operator/values.yaml

```
@@ -153,6 +153,10 @@ admissionWebhooks:
   failurePolicy: Fail
   secretName: ""
 
+  ## Mutating webhooks are called sequentially. In some cases you would like your MutatingWebhooks
+  ## to run in specific order. In such case, add a prefix to all of them to sort alphabetically.
+  orderPrefix: ""
```

```suggestion
  prefix: ""
```

Since this field truly is a prefix to the name I don't think we need to include `order` in the name.

---

maybe "namePrefix"?